File: solutions/beecrowd/2850/2850.py
import sys

for line in sys.stdin:
    clean_input = line.strip()
    if clean_input == 'esquerda':
        print('ingles')
    elif clean_input == 'direita':
        print('frances')
    elif clean_input == 'nenhuma':
        print('portugues')
    else:
        print('caiu')

File: solutions/beecrowd/2850/2850.cpp
#include <cstdint>
#include <iostream>
#include <string>

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(0);

    std::string leg;

    while (getline(std::cin, leg)) {
        if (leg == "esquerda") {
            std::cout << "ingles\n";
        } else if (leg == "direita") {
            std::cout << "frances\n";
        } else if (leg == "nenhuma") {
            std::cout << "portugues\n";
        } else if (leg == "as duas") {
            std::cout << "caiu\n";
        }
    }

    return 0;
}
